app/managers.py

```diff
@@ -0,0 +1,46 @@
+import sqlite3
+from app.models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connection = sqlite3.connect("cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
```

Comment:
Ensure that the `id` field in the database is set to auto-increment. This will allow new actors to be created with a unique `id` automatically.